spec/requests/api_registrations_spec.rb

```diff
@@ -1275,54 +1275,69 @@
   end
 
   describe 'GET #payment_ticket' do
-    let(:competition) { create(:competition, :registration_open, :with_organizer, :stripe_connected) }
-    let(:reg) { create(:registration, :pending, competition: competition) }
     let(:headers) { { 'Authorization' => fetch_jwt_token(reg.user_id) } }
 
-    it 'successfully builds a payment_intent via Stripe API' do
-      get api_v1_registrations_payment_ticket_path(competition_id: competition.id), headers: headers
-      expect(response).to be_successful
-    end
+    context 'stripe connect payment integration' do
```

Comment:
I moved everything that was here into this context block, and added a new context block for manual payment integrations - this makes the diff appear larger/more significant than it is